airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeSqlOperations.java

```diff
@@ -69,9 +69,9 @@ public String createTableQuery(final JdbcDatabase database, final String schemaN
 
   /**
    * Sort of hacky. The problem is that SnowflakeSqlOperations is constructed in the
-   * SnowflakeDestination constructor, but we don't have the JsonNode config until we try to call
-   * check/getSerializedConsumer on the SnowflakeDestination. So we can't actually inject the config
-   * normally. Instead, we just use the singleton object. :(
+   * SnowflakeDestinationOld constructor, but we don't have the JsonNode config until we try to call
```

Comment:
in between refactor, intellij used the tmp class name. will rename it in comment.